package.json

```diff
@@ -19,8 +19,9 @@
     "dotenv": "^16.4.5",
     "express": "^4.18.3",
     "morgan": "^1.10.0",
-    "rimraf": "^5.0.5",
     "pg": "^8.11.3",
+    "rimraf": "^5.0.5",
+    "uuid": "^9.0.1",
```

Comment:
Let's delete this package, we don't generate UUIDs in js code but we do it in the database.
You can do it by running "npm remove uuid"